### ISSUE TITLE
Emacs: nix-mode.el

### ISSUE THREAD

**Issue by Pitometsu** [NixOS/nix#632]
### Font-lock problem
![Emacs screenshot](http://i.imgur.com/OpcDd4p.png)
```nix
      (let inconsolata-lgc  = with pkgs; stdenv.mkDerivation rec {
             name = "inconsolata-lgc-${version}";
             version = "1.2.1";
             src = fetchgit {
               url = "git://github.com/clozed2u/Inconsolata-LGC.git";
               rev = "ec27f1a0f92c6ce63b10b838fc6bafbcc71c9fed";
               sha256 = "06zyqqv1z3ghfc57jbjkzis1nigkhv1k3j7i4qkg5nc7haj9h4l0";
             };
             phases = [ "installPhase" ];
             installPhase = ''
               mkdir -pv "$out/share/fonts/truetype"
	       ls -lR "$src"
	       ls -lR "$out/share/fonts/truetype"
               cp -v "$src"/*.ttf "$out/share/fonts/truetype" # <- Problem here!
             '';
             meta = with stdenv.lib; {
               homepage = http://www.levien.com/type/myfonts/inconsolata.html;
               description = "A monospace font for both screen and print, LGC extension";
               license = licenses.ofl;
               platforms = platforms.all;
             };
           }; in inconsolata-lgc)
```

**Comment by gleber** [comment]
I'm having the same problem, but not enough elisp-fu to fix it.

**Comment by Ericson2314** [comment]
Sorta duplicate: #253

**Comment by Ericson2314** [comment]
This can be closed because of #870

**Comment by Pitometsu** [comment]
Confirm: fixed.
